### ISSUE TITLE
Contact us is not working

### ISSUE THREAD

**Issue by Selenium100** [Yash-srivastav16/Tour-Project#22]
Go to Contact us
Enter your data
click on Send

405 method not allowed error is displayed

<img width="665" alt="image" src="https://github.com/Yash-srivastav16/Tour-Project/assets/48276449/3b0c9f5c-a52e-4d5c-9cc8-bf63d62246df">

**Comment by Yash-srivastav16** [comment]
Yes you are correct as currently we are using local server for this, this needs to be changes to any live server

**Comment by Yash-srivastav16** [comment]
If any one is interested to work on this.?

**Comment by Selenium100** [comment]
Could you please resolve and let me know

**Comment by cpwallah** [comment]
yes i am interested can you assign me this

**Comment by cpwallah** [comment]
@Yash-srivastav16  @Selenium100  i am interested

**Comment by Nithya113** [comment]
I'm interested in taking this issue.

**Comment by harshpalarpwar2002** [comment]
why contact us is not working

**Comment by Yash-srivastav16** [comment]
Any one want to pick this up?
